.drone.starlark

```diff
@@ -0,0 +1,1073 @@
+config = {
+	'app': 'files_antivirus',
+	'rocketchat': {
+		'channel': 'builds',
+		'from_secret': 'private_rocketchat'
+	},
+
+	'branches': [
+		'master'
+	],
+
+	'appInstallCommand': 'make',
+
+	'codestyle': {
+		'ordinary' : {
+			'phpVersions': [
+				'7.0',
+				'7.1',
+				'7.2',
+				'7.3',
+			],
+		},
+	},
+
+	'phpunit': {
+		'allDatabases' : {
+			'phpVersions': [
+				'7.0',
+			]
+		},
+		'reducedDatabases' : {
+			'phpVersions': [
+				'7.1',
+				'7.2',
+				'7.3',
+			],
+			'databases': [
+				'sqlite',
+				'mariadb:10.2',
+			],
+			'coverage': False
+		},
+	},
+
+	'acceptance': {
+		'webUI': {
+			'suites': {
+				'webUIActivityList': 'webUIActList',
+			},
+			'browsers': [
+				'chrome',
+				'firefox'
+			],
+			'extraApps': [
+				'activity'
+			],
+		},
+		'api': {
+			'suites': [
+				'apiAntivirus'
+			],
+			'databases': [
+				'mariadb:10.2',
+				'mysql:5.5',
+				'mysql:5.7',
+				'postgres:9.4',
+				'oracle'
+			],
+		},
+		'api-objectstore': {
+			'suites': {
+				'apiAntivirus': 'apiAntivirusObjectstore',
```

Comment:
The name `apiAntivirusObjectstore` gets used as part of the matrix entry name in the drone.
So having a unique name here makes the drone matrix entry names unique.
Otherwise you end up with duplicate matrix entry names like `apiAntivirus-master-mariadb10.2-php7.0`